### PR TITLE
bug vs feature

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,10 @@ Projects can be used to group and preview issues more nicely. [ODA Platform proj
 To avoid duplication, **milestones** can be seen as primary indication of the issue/PR assignment to the release, and **project** is a complementary presentation feature.
 
 **Labels**, on the other hand, describe different aspects of issues (documentation, bug, etc). Details of the labels purpose should be put in the label descriptions.
+We have the following labels, among others:
+
+* **Bug** is something that violates feature which we assume to work. Bug needs to be addressed ASAP, sometimes with a hotfix, or at least included in "Known Issues" (bug included in known issues should be considered a future feature). 
+* **Feature** is something we know we did not have but would like to have. Feature development is to be scheduled according to the requirements and availability.
```
Comment (burnout87) [inline README.md:109]: Can't a feature also be something we already provide?

Comment (volodymyrss) [reply README.md:109]: This is not about "feature" as a word, this is specifically "feature" **label**. Label is assigned to PR or issue - something we want to add.

Comment (burnout87) [reply README.md:109]: ok thanks for the clarification

**Review by burnout87 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (volodymyrss) [reply README.md]: here to increase clarity

**Pull request merged**